README.md

```diff
@@ -12,71 +12,93 @@ Takeaway Challenge
       '. '' .'    \:.....:--'.-'' .'
        ':..:'                ':..:'
 
- ```
+```
 
-Instructions
--------
+The task was to build a program that displayed a menu, adds things to a basket, displays a basket, completes an order and sends a text message confirming the order.
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+## Getting Started
 
-Task
------
+```sh
+$ git clone https://github.com/saypop/takeaway-challenge.git
+$ cd takeaway-challenge
+```
+Then see Deployment below.
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+### Prerequisites
 
+You'll need ruby installed on your system.
+```sh
+$ brew install ruby
 ```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+And you'll need Twilio.
+```sh
+$ gem 'twilio-ruby'
+```
```

Comment:
this is in your Gemfile, so you should say so! you can install everything with `bundle install`